fmn/rules/ci.py

```diff
@@ -0,0 +1,234 @@
+from fmn.lib.hinting import hint, prefixed as _
+
+# Basic building steps
+
+@hint(categories=['ci'])
+def ci_catchall(config, message):
+    """ All CI events
+
+    Adding this rule will indiscriminately match notifications of all types
+    from `ci <https://admin.fedoraproject.org/pkgdb>`_, i.e. new build queued,
```

Comment:
Same url appears elsewhere in the module.  Maybe use https://fedoraproject.org/wiki/FedoraAtomicCI/pipeline ?

You may always want to replace the vague "ci" with the more specific "the Fedora Atomic CI pipeline".